docker-compose/run_selenium_tests_local_keybase.sh

```diff
@@ -0,0 +1,217 @@
+#!/bin/bash
+
+# Run the selenium tests in docker
+# 
+# NOTE:
+#
+#   1. You must be logged in to Keybase and have the BB2 team file system mounted.
+#
+#   2. You must also be connected to the VPN.
+#
+# SETTINGS:  You may need to customize these for your local setup.
+
+HOSTNAME_URL="http://bb2:8000"
+
+KEYBASE_ENV_FILE="team/bb20/infrastructure/creds/ENV_secrets_for_local_integration_tests.env"
+KEYBASE_CERTFILES_SUBPATH="team/bb20/infrastructure/certs/local_integration_tests/fhir_client/certstore/"
+
+BB20_TEAM_VAULT_PASSFILE="/cygdrive/k/team/bb20/infrastructure/creds/vault_pw.txt"
+BB20_DEPLOYMENT_VAULTFILE="../bluebutton-web-deployment/vault/env/test/vault.yml"
+BB20_ENABLE_REMOTE_DEBUG=True
+
+DJANGO_FHIR_CERTSTORE="/certstore"
+CERTSTORE_TEMPORARY_MOUNT_PATH="/tmp/certstore"
+CERT_FILENAME="client_data_server_bluebutton_local_integration_tests_certificate.pem"
+KEY_FILENAME="client_data_server_bluebutton_local_integration_tests_private_key.pem"
+
+# Backend FHIR server to use for selenium tests with FHIR requests:
+FHIR_URL="https://prod-sbx.bfd.cms.gov"
+
+# List of tests to run. To be passed in to runtests.py.
+TESTS_LIST="apps.integration_tests.selenium_tests.SeleniumTests"
+
+
+# Echo function that includes script name on each line for console log readability
+echo_msg () {
+  echo "$(basename $0): $*"
+}
+
+# main
+echo_msg
+echo_msg RUNNING SCRIPT:  ${0}
+echo_msg
+
+DJANGO_USER_ID_SALT=$(ansible-vault view --vault-password-file=${BB20_TEAM_VAULT_PASSFILE} ${BB20_DEPLOYMENT_VAULTFILE} | grep "^vault_env_django_user_id_salt" | awk '{print $2}')
+DJANGO_USER_ID_ITERATIONS=$(ansible-vault view --vault-password-file=${BB20_TEAM_VAULT_PASSFILE} ${BB20_DEPLOYMENT_VAULTFILE} | grep "^vault_env_django_user_id_iterations" | awk '{print $2}')
+
+DJANGO_PASSWORD_HASH_ITERATIONS=$(ansible-vault view --vault-password-file=${BB20_TEAM_VAULT_PASSFILE} ${BB20_DEPLOYMENT_VAULTFILE} | grep "^vault_env_django_password_hash_iterations" | awk '{print $2}')
+DJANGO_SLSX_CLIENT_ID=$(ansible-vault view --vault-password-file=${BB20_TEAM_VAULT_PASSFILE} ${BB20_DEPLOYMENT_VAULTFILE} | grep "^vault_env_slsx_client_id" | awk '{print $2}')
+DJANGO_SLSX_CLIENT_SECRET=$(ansible-vault view --vault-password-file=${BB20_TEAM_VAULT_PASSFILE} ${BB20_DEPLOYMENT_VAULTFILE} | grep "^vault_env_slsx_client_secret" | awk '{print $2}')
```

Comment:
Couple of things not great about this:
- We're duplicating setting a couple of these variables when sourcing the integration tests env file (due to being copied from the integration tests bash script?)
- These variables are not set safely (not protected by set -u below, also not checked for existence via if [ -z ] test) so they can break things later down the line
- We're relying on the test ENV vault which is meant for AWS TEST env, can't always guarantee this will be right for local development in the future

I would recommend putting all new secrets needed in the integration test .env file in keybase and sourcing them all from there as the integration test bash script does, and removing all dependencies on ansible vault.